Multiple vectors docs (#19)
qdrant/docs: qdrant/v0.10.x/collections.md

- `shard_number` - which defines how many shards the collection should have. See [distributed deployment](../distributed_deployment#sharding) section for details.
- `on_disk_payload` - defines where to store payload data. If `true` - payload will be stored on disk only. Might be useful for limiting the RAM usage in case of large payload.

*Available since v0.10.0*

> There is support for multiple vectors per record. This feature allows for content multiple vector storages per collection. To separate different vectors in one record, each vector should have a unique name that should be defined while collection creation at once. Each named vector in this mode has its own distance and size:

Reviewer: I find this paragraph a bit heavy, trying to make this read a bit smoother.

```suggestion
It is possible to have multiple vectors per record. This feature allows for multiple vector storages per collection. To separate different vectors in one record, each vector should have a unique name that should be defined when creating the collection. Each named vector in this mode has its own distance and size:
```

Author: Thanks a lot!
